nms/app/fbcnms-projects/magmalte/app/views/events/EventsTable.js

```diff
@@ -250,25 +338,55 @@ export default function EventsTable({
       )}
       {sz === 'lg' && (
         <div className={classes.dashboardRoot}>
-          <ActionTable
-            title="Events"
-            titleIcon={MyLocationIcon}
-            data={eventRows}
-            columns={[
-              {title: 'Timestamp', field: 'ts', type: 'datetime'},
-              {title: 'Event Type', field: 'eventType'},
-              {title: 'Event Description', field: 'eventDescription'},
-              {
-                title: 'More Details',
-                field: 'eventDescription',
-                render: rowData => <ExpandEvent rowData={rowData} />,
-              },
-            ]}
-            options={{
-              actionsColumnIndex: -1,
-              pageSizeOptions: [10, 20],
-            }}
-          />
```

Comment:
Why did you switch off the ActionTable?

Reply:
I didn't turn it off, i moved it under the grid.